File: corpus/2017q1/_p2q3_ok/_check/B;0604.java

import java.util.Scanner;
public class P203 {
static int L=0,C=0;
    public static void main(String[] args) {
    Scanner entrada = new Scanner(System.in);

	System.out.println ("Insira as dimensões da matriz:");
	L = entrada.nextInt();
        C = entrada.nextInt();
        int A[][] = new int [L][C];
        int B[][] = new int [L][C];

	LER(A,L,C);

	IMPRIME(A,L,C);

	NOVAMATRIZ(A,L,C);

	IMPRIME(B,L,C);
	}
	public static void LER(int A[][], int L, int C){
	Scanner entrada = new Scanner(System.in);
        int i,j, maior=0;
	System.out.println("Insira os elementos da matriz");
		for(i=0;i<L;i++){
			for(j=0;j<C;j++){
			A[i][j] = entrada.nextInt();
			}
		}

	}
    public static void NOVAMATRIZ(int A[][], int L, int C){
	int B[][] = new int [L][C]; 
        int i,j, maior=0;
        for(i=0;i<L/2;i++){
            for(j=0;j<C/2;j++){
                B[i][j]=0;
            }
        }
	 for(i=L/2;i<L;i++){
            for(j=C/2;j<C;j++){
                B[i][j]=0;
            }
        }
         for(i=0;i<L/2;i++){
            for(j=C/2;j<C;j++){
                B[i][j]=0;
            }
        }

         for(i=L/2;i<L;i++){
            for(j=0;j<C/2;j++){
                B[i][j]=A[i][j];
            }
        }

    }
    public static void IMPRIME(int A[][], int L, int C){
	int i,j;
        System.out.println("Elementos da matriz");
	for (i=0;i<L;i++){
	for(j=0;j<C;j++){
		System.out.println("\n"+ A[i][j]);		
				}
			}

	}
}

File: corpus/py/B;0604.py
static int L=0,C=0
print ("Insira as dimensões da matriz:")
L = int(input())
C = int(input())
LER(A,L,C)
IMPRIME(A,L,C)
NOVAMATRIZ(A,L,C)
IMPRIME(B,L,C)
def LER(int A[][], int L, int C):
    int i,j, maior=0
    print("Insira os elementos da matriz")
    for i=0 in range(0, L):
        for j=0 in range(0, C):
def NOVAMATRIZ(int A[][], int L, int C):
    int i,j, maior=0
    for i=0 in range(0, L/2):
        for j=0 in range(0, C/2):
    for i=L/2 in range(L/2, L):
        for j=C/2 in range(C/2, C):
    for i=0 in range(0, L/2):
        for j=C/2 in range(C/2, C):
    for i=L/2 in range(L/2, L):
        for j=0 in range(0, C/2):
def IMPRIME(int A[][], int L, int C):
    int i,j
    print("Elementos da matriz")
    for i=0 in range(0, L):
        for j=0 in range(0, C):
            print("\n"+ A[i][j])